setup.py

```diff
@@ -162,6 +162,8 @@ def is_excluded_dir(dir):
         install_requires=REQUIRES,
         tests_require=TESTS_REQUIRE,
         extras_require=EXTRAS_REQUIRE,
-        packages=find_packages(exclude=EXCLUDE_PATTERNS),
+        packages=find_packages(
+            exclude=["workbench*", "shoop_tests*"] + EXCLUDE_PATTERNS
+        ),
```

Comment:
This will break tox, since it needs workbench and shoop_tests for running the tests. Can we just rename workbench to shoop_workbench and let it get installed and the shoop_tests too?